### DISCUSSION TITLE
11/21 블로그 1회차

### DISCUSSION THREAD

**Discussion by hansol-woff** [Orm-camp/blog-feedback#39] [동글동글 3기 피드백]
https://velog.io/@incolore9/디자이너의-개발자-입문기-첫-달-회고

조금 가볍게 소감 위주로만 작성했습니다..! 피드백 부탁드려요.
부랴부랴 썼는데 차차 썸네일이나 중간중간 이미지도 넣어서 수정하겠습니다.

**Comment by baaanjy** [comment]
공감가는 내용이 많네요.. 특히 잠..!!! 🥲

**Comment by kimjigyun** [comment]
정말요 .. 잠자는것도 잘 해야됩니당 !
열심히 하시는 모습을 보니까 저도 노력해야겠다 생각했습니다 !

**Comment by Orm-camp** [comment]
한 편의 수필 같았던 한솔님의 회고 글 잘 읽었습니다!
저는 한솔님의 학습 여정을 옆에서 함께 보고 있는 입장이라 이 글을 쭈욱 읽는 것에 무리가 없었는데요.

처음 마주하는 분들을 위해서는 조금 더 친절한 장치들이 필요하다고 생각이 들었어요!

- 불렛 포인트를 활용해서 중요한 내용들을 끊어서 작성하기
- 문단 간의 사이를 더 늘려서 읽기 편하게 하기
위와 같은 장치가 마련된다면 더더욱 읽기 좋지 않을까 하는 생각이 듭니다 👍🏻 

한솔님의 다음 글도 기다려져요!

**Comment by ksj0621** [comment]
공감 가는 부분이 많아 후루룩 읽히는 글이었습니다! 그래도 정말 성장하시는 모습이 팀 회고 시간에 가끔 뵙는 저도 느껴질 정도이니 잘하고 계신다고 스스로 다독여 주심 좋을 것 같아요!
